dir/path.go

```diff
@@ -120,3 +153,18 @@ func (p *PathManager) CachedSignatureStoreDirPath() string {
 	checkError(err)
 	return path
 }
+
+func getPathForWrite(user UnionDirFS, system UnionDirFS, item ...string) string {
```

Comment:
since 'item' could be more than one,  'items' might be better.

Reply:
Done.